api/Insight_API_documentation.md

```diff
@@ -0,0 +1,1200 @@
+                  "hex": "6a24c0f1e0de5187474c87e8ee9b323bda84562d675ea0126c1e0000000000000000b3270400",
+                  "asm": "OP_RETURN c0f1e0de5187474c87e8ee9b323bda84562d675ea0126c1e0000000000000000b3270400",
+                  "type": "nulldata"
+               },
+               "spentTxId": null,
+               "spentIndex": null,
+               "spentHeight": null
+            },
+            {
+               "value": 0,
+               "n": 1,
+               "scriptPubKey": {
+                  "hex": "6a06010005000000",
+                  "asm": "OP_RETURN 010005000000",
+                  "type": "nulldata"
+               },
+               "spentTxId": null,
+               "spentIndex": null,
+               "spentHeight": null
+            },
+            {
+               "value": 0.01196299,
+               "n": 2,
+               "scriptPubKey": {
+                  "hex": "bb76a914fdf7e454442af6795aad73bc3dd844b27b289c5e88ac",
+                  "asm": "OP_SSGEN OP_DUP OP_HASH160 fdf7e454442af6795aad73bc3dd844b27b289c5e OP_EQUALVERIFY OP_CHECKSIG",
+                  "addresses": [
+                     "Dsp7mQo6LRUasH8p6socnmq1iqDU2W4HXCp"
+                  ],
+                  "type": "stakegen"
+               },
+               "spentTxId": null,
+               "spentIndex": null,
+               "spentHeight": null
+            },
+            {
+               "value": 95.87354,
+               "n": 3,
+               "scriptPubKey": {
+                  "hex": "bb76a914b7a0fadc16c16c1d6607b3e8ebc3ff4fc214a87488ac",
+                  "asm": "OP_SSGEN OP_DUP OP_HASH160 b7a0fadc16c16c1d6607b3e8ebc3ff4fc214a874 OP_EQUALVERIFY OP_CHECKSIG",
+                  "addresses": [
+                     "DshhqzchyKUzPuZ8QQWwuepCg7qzBd8yUrt"
+                  ],
+                  "type": "stakegen"
+               },
+               "spentTxId": null,
+               "spentIndex": null,
+               "spentHeight": null
+            }
+         ],
+         "blockhash": "0000000000000000bffc01c7350dd0a58619e340ab4d75d8d3ff42e201d67503",
+         "blockheight": 272308,
+         "confirmations": 7501,
+         "time": 1536307801,
+         "blocktime": 1536307801,
+         "valueOut": 95.885506,
+         "size": 420,
+         "valueIn": 95.885506,
+         "fees": 1.0E-8
+      }
+   ]
+}
+```
+
+<br/>
+
+## Status and Utility
+
+Methods that provide utilities or relay network status.
+
+### /sync/
+
+**URL:**  ```GET /sync/```
+
+**Description:** Syncs with historical blockchain data. **REVIEW WORDING**
+
+
+**Request Example:**
+
+```GET /sync```
+
+**Request Response:**
+
+```
+{
+    "status": "finished",
+    "blockChainHeight": 279815,
+    "syncPercentage": 100,
+    "height": 279815,
+    "error": null,
+    "type": "from RPC calls"
+}
+```
+
+<br/>
+
+### /peer/
+
+**URL:**  ```GET /peer/```
+
+**Description:** Retrieves Peer-to-Peer (P2P) data sync status.  **REVIEW WORDING**
+
```

Comment:
the wording is ok.  I'd say leave it.  

dcrdata syncs with dcrd locally.  dcrd has peers, but we don't report that.  All this endpoing does is provide compatiblity with insight.